arxiv/base/routes.py

```diff
@@ -20,7 +20,7 @@
 @blueprint.route('/test', methods=['GET'])
```

Comment:
This seems fine, though could be something more specific (/styleguide)?

Reply:
Good thought, I'll change it! I had repurposed an old route, but you're right that it isn't descriptive of the current function.